src/intl/messages/index.ts

```diff
@@ -1,7 +1,9 @@
 import en from './en.json'
 import fr from './fr.json'
+import oc from './oc.json'
```

Comment:
```suggestion
import ca from './ca.json'
```

Reply:
Hello no, it's Occitan, of wihch Provençal is a dialect ;)

Reply:
Oh ok! I didn't know `navigator` was able to support this. Your browser actually can return the `oc` code ?

Reply:
sure, I have my Firefox in Occitan. This is my accepted_language header: oc,fr,fr-FR,en-US,en

Reply:
Great so! I'll merge it but not instantly deploy it because I was doing a refactor. It will be available asap. Thanks!